com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/ProtoBuilder.java

```diff
@@ -25,8 +27,37 @@
 
     private ProtoParams protoParams;
 
+    private static Map<String, Class<? extends GeneratedMessage>> extToMessageClass = new HashMap<String, Class<? extends GeneratedMessage>>();
```

Comment:
This mechanic is to allow for a dynamic registration of protobuf message builders, as opposed to the hard coded list we had.